deploy/docker/compose.local.secure.yml

```diff
@@ -13,11 +13,15 @@ services:
       - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
       - "--providers.file.directory=/etc/traefik/dynamic"
       - "--providers.file.watch=true"
+      - "--certificatesresolvers.manual.acme.tlschallenge=true"
+      - "--certificatesresolvers.manual.acme.email=your-email@domain.com"
+      - "--certificatesresolvers.manual.acme.storage=/etc/traefik-certs/acme.json"
     ports:
       - "80:80"
       - "443:443"
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - "/etc/letsencrypt:/etc/letsencrypt:ro"
```

Comment:
letsencrypt does not work for localhost. That is why we are using mkcert in [secure localhost](https://into-cps-association.github.io/DTaaS/version0.7/admin/localhost-secure.html). Please use [secure production](https://into-cps-association.github.io/DTaaS/version0.7/admin/host.html) installation to integrate the container registry.